files-community/Files
# Fix: Fixed issue where new tab icon disappears when window is smaller than 640px

**Resolved / Related Issues**
   Closes #11797 

**Changes**
- Now `New Tab` button is always visible
- Fixes a bug where you couldn't close tabs (using `x` button) when window was smaller than 640px

**Validation**
How did you test these changes?
   1. **NOTE** To test changes you have to build Files as release or comment out line 128 of `DragZoneHelper.cs` (XAML toolbar may interfere with taps)
   2. Resize the window so that it it smaller than 640px
   3. Try to create a tab using the `+` button
